extras/docker/base/Dockerfile

```diff
@@ -6,7 +6,10 @@ MAINTAINER Counterparty Developers <dev@counterparty.io>
 RUN apt-get update && apt-get install -y apt-utils ca-certificates wget curl git mercurial \
     python3 python3-dev python3-pip python3-setuptools python3-appdirs \
     build-essential vim unzip software-properties-common sudo gettext-base \
-    net-tools iputils-ping telnet lynx
+    net-tools iputils-ping telnet lynx nano
+
+RUN wget https://raw.githubusercontent.com/CounterpartyXCP/federatednode/master/extras/jq-linux64
+RUN sudo mv jq-linux64 /usr/bin/jq; sudo chmod +x /usr/bin/jq
```

Comment:
debugging tool that's very helpful for extracting values from json output into bash vars (for tutorials).

pulling it from our own repo so it doesn't rely on external / untrusted source (ubuntu repos contain old version :/)